CHANGELOG.md

```diff
@@ -1,3 +1,104 @@
+# 0.4.4
+
+Breaking changes:
+
+- `Temporal.ZonedDateTime.prototype.toLocaleString` now works without `Intl.DateTimeFormat`, and `Temporal.ZonedDateTime` objects are no longer supported as parameters to `Intl.DateTimeFormat` formatting methods. ([12071fb0], see also [Upstream PR](https://github.com/tc39/proposal-temporal/pull/2479), [Upstream PR 2](https://github.com/tc39/proposal-temporal/pull/2522))
+  - To format using `Temporal.ZonedDateTime`'s time zone, use `Temporal.ZonedDateTime.prototype.toLocaleString`. This method:
+    1. Creates a new `Intl.DateTimeFormat` instance in the same time zone and calendar as the `Temporal.ZonedDateTime` instance
+    1. Creates a `Temporal.Instant` from the `Temporal.ZonedDateTime` instance.
+    1. Formats that `Temporal.Instant` instance using the created `Intl.DateTimeFormat` instance.
+  - To format in a time zone that does not match the time zone of the `Temporal.ZonedDateTime` instance:
+    1. Create a new `Temporal.ZonedDateTime` instance in the right time zone using `Temporal.ZonedDateTime.prototype.withTimeZone`.
+       - To get the current system time zone, use `Temporal.Now.timeZoneId()`.
+         Note that caching this value can be very problematic because the system time zone can change during the lifetime of your program (e.g. when a user on a mobile device crosses a time zone boundary, users manipulate their system clock settings manually).
```

Comment:
```suggestion
         Be careful when caching the current system time zone (or an `Intl.DateTimeFormat` instance using that time zone), because the system time zone can change during the lifetime of your program (e.g. when a user on a mobile device crosses a time zone boundary or when a user manually edits time zone settings).
```